weaviate/collection/grpc.py

```diff
@@ -69,11 +70,11 @@ class MetadataQuery:
     explain_score: bool = False
 
 
-@dataclass
-class LinkTo:
+class LinkTo(BaseModel):
```

Comment:
I am wondering if it makes sense to have two classes here?

```
class LinkTo(BaseModel): 
    link_on: str
    properties: "PROPERTIES"
    metadata: MetadataQuery
```

and 

```
LinkToMultiTarget(LinkTo):
    which_collection:str
```

What do you think?

Reply:
I think that's a great idea! It will help to differentiate the two specific use-cases and avoid user error 😁